Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -252,6 +252,7 @@ To install the mask extension in version 4.1.\*, type:
 
    composer require mask/mask:~4.1.0
```

Comment:
Please use an up-to-date version for mask.

Reply:
Not the scope of this PR, but I happily added it :)